.gitignore

```diff
@@ -161,4 +161,6 @@ cython_debug/
 
 **/env/
 **/key.txt
-**/local-auth.json
\ No newline at end of file
+**/local-auth.json
+/app/features/quizzify/tools copy.py
```

Comment:
If its a copy just for local machine and it is no longer used, we should remove the `copy` file altogether